### DISCUSSION TITLE
faqs/numbers/

### DISCUSSION THREAD

**Discussion by giscus** [reubenharry/haskell-docs#48] [Q&A]
# faqs/numbers/

https://haskell-docs.netlify.app/faqs/numbers/

**Comment by cmkay56** [comment]
Perhaps you meant double = (\x -> x + x) :: Double -> Double  |  double n = 10
or
square = (\x -> x * x) :: Double -> Double   | square n = 25

**Reply by reubenharry** [reply]
Good catch! Now updated to `double = (\x -> x + x)`.
